README.md

```diff
@@ -89,6 +89,10 @@ const style = css({
 })
 ```
 
+### Images and other static files
+
+Create a folder called `static` in your project root directory. You can place images and other static assets there. Reference them from JS files with the path to the static directory. For example, from `/pages` use `../static/file-name.jpg`
```

Comment:
I wonder why you'd like to get references of static files from JS. Do you mean to embed reference to DOM like `<img src="/static/file-name.jpg" />` ?